content/docs/latest/installing/cloud/azure.md

```diff
@@ -78,34 +76,66 @@ $ az vm create --name node-1 --resource-group group-1 --admin-username core --us
         <p>The Alpha channel closely tracks the master branch and is released frequently. The newest versions of system
         libraries and utilities are available for testing in this channel. The current version is Flatcar Container Linux {{< param alpha_channel >}}.</p>
         <pre>
-$ az vm image list --all -p kinvolk -f flatcar -s alpha
-[
-  {
-    "offer": "flatcar-container-linux",
-    "publisher": "kinvolk",
-    "sku": "alpha",
-    "urn": "kinvolk:flatcar-container-linux:alpha:2430.0.0",
-    "version": "2430.0.0"
-  }
-]
-$ az vm create --name node-1 --resource-group group-1 --admin-username core --user-data config.ign --image kinvolk:flatcar-container-linux:alpha:2430.0.0
+$ az vm image list --all -p kinvolk -f flatcar -s alpha-gen2 --query '[-1]'
+{
+  "architecture": "x64",
+  "offer": "flatcar-container-linux-free",
+  "publisher": "kinvolk",
+  "sku": "alpha-gen2",
+  "urn": "kinvolk:flatcar-container-linux-free:alpha-gen2:3874.0.0",
+  "version": "3874.0.0"
+}
+$ az vm create --name node-1 --resource-group group-1 --admin-username core --user-data config.ign --image kinvolk:flatcar-container-linux-free:alpha-gen2:3874.0.0
         </pre>
       </div>
     </div>
   </div>
 </div>
 
-You can use both image offers `flatcar-container-linux` and `flatcar-container-linux-free`, the contents are the same.
-The SKU, which is the third element of the image URN, relates to one of the release channels and also depends on whether to use HyperV Generation 1 or 2.
-Generation 1 instance types use the channel names `alpha`, `beta` or `stable` as is; for Generation 2 instance types please append `-gen2` to the channel name, i.e., use one of `alpha-gen2`, `beta-gen2` or `stable-gen2`.
-This means the Gen 2 image URN for the above example for a Stable release becomes `flatcar-container-linux:stable-gen2:2345.3.0`.
+Use the offer named `flatcar-container-linux-free`, there is also a legacy offer called `flatcar-container-linux` with the same contents.
+The SKU, which is the third element of the image URN, relates to one of the release channels and also depends on whether to use Hyper-V Generation 1 or 2 VM.
+Generation 2 instance use UEFI boot and should be preferred, the SKU matches the pattern `<channel>-gen`: `alpha-gen2`, `beta-gen2` or `stable-gen2`.
```

Comment:
```suggestion
Generation 2 instances use UEFI boot and should be preferred, the SKU matches the pattern `<channel>-gen`: `alpha-gen2`, `beta-gen2` or `stable-gen2`.
```

Reply:
i was going to write `instances` but missed the `s`. Types sounds good too :)

Reply:
fixed it